metrics/prometheus/operator/Chart.yaml

```diff
@@ -8,8 +8,8 @@ keywords:
   - Prometheus operator
   - Coralogix
 dependencies:
-  - name: kube-prometheus-stack 
-    version: "41.7.3"
+  - name: kube-prometheus-stack
+    version: "45.30.0"
```

Comment:
Please add comment to readme.

Reply:
where?

Reply:
https://github.com/coralogix/telemetry-shippers/tree/6b19699c92defd6d20c7fcff2519b44225fad2ca/metrics/prometheus/operator

We are about to restructure the docs here to make it more clear. But if you can add note for now it will be helpful. Thx.

Reply:
@oded-dd added, let me know if this is what you meant (my question regarding where was meant to be "where in the Readme file you want it to be added).
thanks